### PR TITLE
Optimized lodash in various modules

### PR DESCRIPTION
**Pull request by luna215**
## Summary
Changed the syntax when importing specific functions from lodash. The syntax I used allows for tree shaking.

## TODO

- [ ] Have tests been written for the new code?
- [ ] Has documentation been written/updated?
- [ ] New dependencies (if any) added to requirements file
- [ ] Add an entry to CHANGELOG.rst
- [ ] Add yourself it AUTHORS.rst if you don't appear there

## Reviewer guidance
I changed the syntax of importing lodash, wherever it was referenced, to allow for tree shaking.

## Issues addressed

Solved the issue of lodash being more than 500kb. Now the average is 100.

## Screenshots (if appropriate)

They're nice. :)

### CODE REVIEW

File: kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/default-language-form/index.vue [hunk 1]
```diff
@@ -61,7 +61,7 @@
   import kButton from 'kolibri.coreVue.components.kButton';
   import onboardingForm from '../onboarding-form';
 
-  import omit from 'lodash/omit';
+  import { omit } afrom 'lodash/omit';
```
Comment (christianmemije) [inline kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/default-language-form/index.vue]: Typo

File: kolibri/plugins/coach/assets/src/views/class-list-page/index.vue [hunk 2]
```diff
@@ -37,7 +37,7 @@
 <script>
 
   import * as constants from '../../constants';
-  import orderBy from 'lodash/orderBy';
+  import { orderBy } from 'lodash/orderBy';
```
Comment (rtibbles) [inline kolibri/plugins/coach/assets/src/views/class-list-page/index.vue]: Does this make a difference to the import?

File: kolibri/plugins/style_guide/assets/src/views/shell/component-docs.vue [hunk 3]
```diff
@@ -81,7 +81,7 @@
 <script>
 
   import escodegen from 'escodegen';
-  import CamelCase from 'lodash/camelCase';
```
Comment (rtibbles) [inline kolibri/plugins/style_guide/assets/src/views/shell/component-docs.vue]: I wonder if this import statement was originally deliberately ironic? (CamelCase imported not in camelCase).

File: kolibri/plugins/coach/assets/src/views/class-list-page/index.vue (hunk 2 above)
Comment (luna215) [reply kolibri/plugins/coach/assets/src/views/class-list-page/index.vue]: Not entirely sure, but it optimized the setup wizard module by a lot, so I stuck to this convention.

**Comment by rtibbles** [comment]
Looks like some of these changes are happening to files that were since modified. And now this is causing linting to fail.

**Comment by rtibbles** [comment]
May be best to start fresh from latest develop to do this.

**Pull request closed without merging**